src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -186,6 +175,38 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
         unmountFn();
       },
 
+      getWidgetRenderState(renderState, { results, state }) {
+        function getItems() {
+          if (!results) {
+            return [];
+          }
+
+          const [{ name: facetName }] = state.hierarchicalFacets;
+
+          const facetValues = results.getFacetValues(
+            facetName,
+            {}
+          ) as SearchResults.HierarchicalFacet;
+          const data = Array.isArray(facetValues.data) ? facetValues.data : [];
+          const items = transformItems(shiftItemsValues(prepareItems(data)));
+
+          return items;
+        }
+
+        const items = getItems();
+
+        return {
+          ...renderState,
+          breadcrumb: {
+            canRefine: items.length > 0,
+            createURL: connectorState.createURL,
+            items,
+            refine: connectorState.refine,
```

Comment:
createURL & refine aren't cached like in clearRefinement, is that expected?

Reply:
I kept it like it was in this PR, but it's not expected. There are multiple connectors that don't keep the same reference. We should fix this and automate some tests run on all widgets.